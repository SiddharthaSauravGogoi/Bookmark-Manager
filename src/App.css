h1{
    text-transform: uppercase;
    text-align: center;
    font-weight: 600;
}