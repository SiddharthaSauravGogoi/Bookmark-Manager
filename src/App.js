import React from 'react';
import './App.css';

function App() {
  return (
    <div className="App">
      Bookmark manager
    </div>
  );
}

export default App;
