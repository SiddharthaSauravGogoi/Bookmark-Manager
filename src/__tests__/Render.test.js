import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import App from '../App';

const mockStore = configureMockStore();
const store = mockStore({});

const wrapped = shallow(
  <Provider store={store}>
    <App />
  </Provider>,
);

describe('Title', () => {
  it('should render the Title Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
