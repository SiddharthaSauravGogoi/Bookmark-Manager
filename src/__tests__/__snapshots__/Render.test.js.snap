// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render the Title Component correctly 1`] = `ShallowWrapper {}`;
